tests/test_client.py

```diff
@@ -265,6 +301,58 @@ def test_score_with_timeout_param_ok(self):
             assert(response.body['scores']['content_abuse']['score'] == 0.14)
             assert(response.body['scores']['payment_abuse']['score'] == 0.97)
 
+    def test_get_user_score_ok(self):
+        """Test the GET /{version}/users/{userId}/score API, i.e. client.get_user_score()
+        """
+        test_timeout = 5
+        mock_response = mock.Mock()
+        mock_response.content = USER_SCORE_RESPONSE_JSON
+        mock_response.json.return_value = json.loads(mock_response.content)
+        mock_response.status_code = 200
+        mock_response.headers = response_with_data_header()
+        with mock.patch('requests.get') as mock_get:
+            mock_get.return_value = mock_response
+            response = self.sift_client.get_user_score('12345', test_timeout)
```

Comment:
Optional: do you also want to test specifying abuse types?

Reply:
Sure, added more tests.